### PR TITLE
Enforce Python version 3.5 and above

### PR DESCRIPTION
**Pull request by danyeaw**
Signed-off-by: Dan Yeaw <dan@yeaw.me>

### CODE REVIEW

File: beekeeper.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ pull_request:
                 name: Python lint checks
                 task: beefore
     - smoke-test:
-        task: py34
-        name: Smoke build (Python 3.4)
+        task: py35
+        name: Smoke build (Python 3.5)
     - full-test:
         subtasks:
             - py3.5:
```
Comment (freakboy3742) [inline beekeeper.yml]: No need to reproduce the 3.5 test here.

File: README.rst [hunk 2]
```diff
@@ -5,7 +5,7 @@
 Waggle
 ======
 
-.. image:: https://img.shields.io/pypi/pyversions/waggle.svg
```
Comment (freakboy3742) [inline README.rst:8]: What's the reason for this change?

**Review by freakboy3742 — CHANGES_REQUESTED** [review]

File: README.rst (hunk 2 above)
Comment (danyeaw) [reply README.rst:8]: The shield was changed so that it shows Python 3.5, 3.6, 3.7.

**Comment by danyeaw** [comment]
I'm not sure I understand why these builds are failing. It looks like we successfully pip install boto3 and botocore 1.8.50. Then Beekeeper runs python setup.py test which is the last line of comb's run.sh but waggle doesn't look like it has any tests in the tests directory. Then beekeeper installs botocore 1.7.48 which is a pinned version that causes errors. I don't know where this pinned version came from though. Beekeeper has version 1.4.4 of boto3 pinned in the requirements.txt which installs version 1.5.95 of botocore. I also don't see it in waggle or comb.

**Comment by danyeaw** [comment]
Closing since waggle will be deprecated in the near future.
